### PR TITLE
Add Ansible Tower integration

### PR DESCRIPTION
**Pull request by calebhailey**
Including a dashboard!

### CODE REVIEW

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+[plugins]: https://docs.sensu.io/sensu-go/latest/plugins/
+[metrics]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/metrics/
+[pipeline]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/pipelines/
+[handler]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/handlers/
+[secret]: https://docs.sensu.io/sensu-go/latest/operations/manage-secrets/secrets/
+[secrets]: https://docs.sensu.io/sensu-go/latest/operations/manage-secrets/secrets/
+[tokens]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/tokens/
+[handler-templating]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/handler-templates/
+[sensu-plus]: https://sensu.io/features/analytics
+[{{dashboard-link}}]: #
```
Comment (jspaleta) [inline integrations/ansible/ansible-tower-remediation/README.md:116]: what this?

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+1. Configure Secrets Management
+
+   This integration requires the following [Sensu Secrets][secrets]:
+
+   - `ansible_tower_host` (hostname or IP address of the Ansible Tower API)
+   - `ansible_tower_token` (Ansible Tower application OAuth token)
+
+   _NOTE: this integration creates one or more Sensu Secrets using the "env" provider. The corresponding environment variables need to be set on every sensu-backend in the Sensu deployment. To add the environment variables, please modify `/etc/default/sensu-backend` or `/etc/sysconfig/sensu-backend` and restart the sensu-backend service(s)._
+
+1. Add a `io.sensu.ansible.config.actions` ("remediation action") annotation to one or more checks.
```
Comment (jspaleta) [inline integrations/ansible/ansible-tower-remediation/README.md:45]: use annotation documentation reference here?

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `ansible-tower` ([pipeline])
+* `ansible-tower` ([handler])
+* `ansible_tower_host` ([secret])
+* `ansible_tower_token` ([secret])
+* `sensu/sensu-ansible-handler` ([asset])
```
Comment (jspaleta) [inline integrations/ansible/ansible-tower-remediation/README.md:15]: version the asset name

**Review by jspaleta — CHANGES_REQUESTED** [review]
Add initial CHANGELOG.md as well

File: integrations/ansible/ansible-tower-remediation/README.md (hunk 1 above)
Comment (calebhailey) [reply integrations/ansible/ansible-tower-remediation/README.md:116]: Removed.

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+<!-- List of all metrics or events collected by this integration. -->
+
+This integration does not produce any [metrics].
+
+## Alerts
+
+<!-- List of all alerts generated by this integration. -->
+
+This integration does not produce any events that should be processed by an alert or incident management [pipeline].
```
Comment (jspaleta) [inline integrations/ansible/ansible-tower-remediation/README.md:89]: Maybe use some text in the Alerts section pointing out this pipeline integration is meant to process events?  And the by default there are no filters in the pipeline

**Review by jspaleta — APPROVED** [review]
Take a look at inline comment about readme addition for Alert section.  But other than that approved

File: integrations/ansible/ansible-tower-remediation/README.md (hunk 4 above)
Comment (calebhailey) [reply integrations/ansible/ansible-tower-remediation/README.md:89]: I think the annotation requirement is covered in the setup section, and remediation actions are specifically not alerts. I think we should leave this as-is.

**Pull request merged**